source/nodejs/adaptivecards-controls/src/constants.ts

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-export const KEY_TAB = 9;
-export const KEY_ENTER = 13;
-export const KEY_ESCAPE = 27;
-export const KEY_SPACE = 32;
-export const KEY_UP = 38;
-export const KEY_DOWN = 40;
-export const KEY_DELETE = 46;
+export const KEY_TAB = "Tab";
```

Comment:
These are exported constants. Any code that currently uses them will be broken. We have to decide whether that's OK or not.

On one hand, it is unlikely that this will affect a lot of applications; on the other hand, as a general rule we just don't introduce breaking changes unless they're absolutely necessary.

Given they are not absolutely necessary here, I think we should keep the old constants as is and introduce the new ones. We can use the `<name>Key` naming convention: `export const TabKey = "Tab";`